doc/source/user_guide/session/launching_ansys_fluent.rst

```diff
@@ -300,6 +300,50 @@ can be launched explicitly using :func:`create_launcher() <ansys.fluent.core.lau
   >>> pim_solver_session = pim_solver_launcher()
 
 
+Launching Fluent in Container Mode with Docker Compose or Podman Compose
```

Comment:
```suggestion
Launching Fluent in container mode with Docker Compose or Podman Compose
```
@mkundu1 It doesn't appear that "Container Mode" is a proper noun, so I'm suggesting initial lowercase.

Reply:
Done. Thank you.